site/scss/catalog/_docs.scss

```
@@ -108,7 +297,70 @@
+  grid-template-columns: 1fr;
+}
+
+[data-moo-catalog-view="list"] .moo-catalog__showcase-card {
+  min-height: 0;
+}
+
+[data-moo-catalog-view="list"] .moo-catalog__showcase-card .card-body {
+  display: flex !important;
+  flex-direction: row;
+  align-items: center;
+  // Bootstrap's .gap-4 utility on the card body compiles to
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)